facts/get_facts.sh

```diff
@@ -65,7 +65,7 @@ case "${osfamily}" in
   wget "http://yum.puppetlabs.com/puppet7-release-el-${operatingsystemmajrelease}.noarch.rpm" -O /tmp/puppet7-release.rpm
   if test -f /tmp/puppet7-release.rpm; then
     rpm -ivh /tmp/puppet7-release.rpm
-    for puppet_agent_version in 7.5.0 7.6.1 7.12.0; do
+    for puppet_agent_version in 7.5.0 7.6.1 7.17.0; do
```

Comment:
can you keep the 7.12 and add 7.17? maybe we should check which puppet updates also contain facter updates and add them as well 🤔

Reply:
For the record sake, 7.12 and 7.17 both provide facter 4.2 and we don't differentiate at the 4.2.x level right now.

I also checked the yum.puppetlabs.com package lists and everything that has a 7.12 agent also has a 7.17 agent.